[

{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#CVE-2020-36222", "@type": ["http://www.w3.org/2002/07/owl#NamedIndividual", "http://docs.oasis-open.org/cti/ns/stix#Vulnerability"], "http://docs.oasis-open.org/cti/ns/stix/core#description": [{"@value": "A flaw was discovered in OpenLDAP before 2.4.57 leading to an assertion failure in slapd in the saslAuthzTo validation, resulting in denial of service."}], "http://docs.oasis-open.org/cti/ns/stix/core#modified": [{"@value": "2021-03-13T20:12:36.146086Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#created": [{"@value": "2021-03-13T20:12:36.146086Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#spec_version": [{"@value": "2.1"}], "http://docs.oasis-open.org/cti/ns/stix/core#id": [{"@value": "vulnerability--98a5eed1-2036-4eb9-bee9-6d5a95aa7696"}], "http://docs.oasis-open.org/cti/ns/stix/core#name": [{"@value": "CVE-2020-36222"}], "http://docs.oasis-open.org/cti/ns/stix/core#created_by_ref": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a"}], "http://docs.oasis-open.org/cti/ns/stix/marking#object_marking_refs": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"}]}

]